large_image/tilesource/__init__.py

```diff
@@ -31,14 +31,15 @@ def isGeospatial(path):
         ds = gdal.Open(path, gdalconst.GA_ReadOnly)
     except Exception:
         return False
-    if ds.GetGCPs() and ds.GetGCPProjection():
-        return True
-    if ds.GetProjection():
-        return True
-    if ds.GetGeoTransform(can_return_null=True):
-        return True
-    if ds.GetDriver().ShortName in {'NITF', 'netCDF'}:
-        return True
+    if ds:
```

Comment:
In the conda installation, GDAL returns a dataset handle even when the file isn't found.  In the pip installation, it raises an exception.  Both have gdal 3.4.0.